Workbooks/CosmosDb Mongo vCore/Overview/Overview.workbook

```diff
@@ -13,13 +13,15 @@
             "label": "Azure Cosmos DB",
             "type": 5,
             "isRequired": true,
-            "value": "value::1",
+            "isGlobal": true,
```

Comment:
this change to make this hidden resource param global probably isn't correct or necessary?  nothing else "downstream" in the template modifies this resource param, does it?